### DISCUSSION TITLE
Firebase Data Connect Support?

### DISCUSSION THREAD

**Discussion by foratbahrani** [invertase/react-native-firebase#7799] [Ideas]
Hi!

Do you have any timeline for adding the new [Data Connect](https://firebase.google.com/docs/data-connect) feature in react native?
It's a game changer.

Thanks.

**Comment by joaqo** [comment]
Would be great.

**Comment by jolindroth** [comment]
I'm also excited for this. Expecting this to be a top request as we transition into public preview!
